components/dashboard/sections/reports/preview/report-builder/predefined-groups.ts

```diff
@@ -0,0 +1,167 @@
+import { ExpenseType, TransactionKind, TransactionType } from '../../../../../../lib/graphql/types/v2/graphql';
+
+import { Group, ReportSection } from '../types';
+
+import { checkGroupsMutualExclusivity } from './check-groups';
+
+// TODO: Internationalization before launching preview feature as default
+export const predefinedGroups: Group[] = [
```

Comment:
Is this ready to be translated or we still want to wait?

Reply:
I think there's a likelihood that we'll reconsider the predefined groups in this cycle so this can wait imo (at the latest until we take it out of preview or make it opt-out)